sentinelhub/config.py

```diff
@@ -215,6 +215,11 @@ def save(self):
         if is_changed:
             self._instance.save_configuration()
 
+    def copy(self):
```

Comment:
Why not just use `copy.copy`?

Reply:
Due to a complex internal structure of `SHConfig` I'm always worried that at some point in the future `copy.copy` won't work correctly. Now that we have a method, which is unit-tests, we can just use this.